sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -119,49 +130,53 @@ Once completed, set the values of the client ID, tenant ID, and client secret of
 
 ```python
 from azure.identity import DefaultAzureCredential
-from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.formrecognizer import DocumentAnalysisClient
 credential = DefaultAzureCredential()
 
-form_recognizer_client = FormRecognizerClient(
+document_analysis_client = DocumentAnalysisClient(
     endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
     credential=credential
 )
 ```
 
 ## Key concepts
 
-### FormRecognizerClient
-`FormRecognizerClient` provides operations for:
+### DocumentAnalysisClient
+`DocumentAnalysisClient` provides operations for analyzing input documents using custom and prebuilt models through the `begin_analyze_document` and `begin_analyze_document_from_url` APIs.
+Use the `model` parameter to select the type of model for analysis.
+
+|Model | Features
+|-|-
 
- - Recognizing form fields and content using custom models trained to recognize your custom forms. These values are returned in a collection of `RecognizedForm` objects.
- - Recognizing common fields from the following form types using prebuilt models. These fields and metadata are returned in a collection of `RecognizedForm` objects.
-    - Sales receipts. See fields found on a receipt [here][service_recognize_receipt].
-    - Business cards. See fields found on a business card [here][service_recognize_business_cards].
-    - Invoices. See fields found on an invoice [here][service_recognize_invoice].
-    - Identity documents. See fields found on identity documents [here][service_recognize_identity_documents].
- - Recognizing form content, including tables, lines, words, and selection marks, without the need to train a model. Form content is returned in a collection of `FormPage` objects.
+|"prebuilt-layout" | Text extraction, selection marks, tables
+|"prebuilt-document" | Text extraction, selection marks, tables, key-value pairs and entities
+|"prebuilt-invoices" | Text extraction, selection marks, tables, and pre-trained fields and values pertaining to English invoices
+|"prebuilt-businessCard" | Text extraction and pre-trained fields and values pertaining to English business cards
+|"prebuilt-idDocument" | Text extraction and pre-trained fields and values pertaining to US driver licenses and international passports
+|"prebuilt-receipt" | Text extraction and pre-trained fields and values pertaining to English sales receipts
+|"{custom-model-id}" | Text extraction, selection marks, tables, labeled fields and values from your custom documents
```

Comment:
Been seeing the discussion about analyze vs extract in this PR, I agree with the distinction. Just noticing that the other prebuilts like businessCards, idDocs, etc are using extract here. Dont feel too strongly about it, but you might want to update those to use analyze?

Reply:
pretty much copying service terminology here, I believe text extraction refers to OCR here